src/posix/Makefile-posix

```diff
@@ -94,7 +94,9 @@ configure_OPTIONS                   = \
 # Platform specific switches
 
 ifeq ($(DAEMON),1)
-configure_OPTIONS              += --enable-posix-daemon
+configure_OPTIONS              += \
+    --enable-posix-daemon         \
+    $(NULL)
```

Comment:
unrelated change?

Reply:
Good catch. It was introduced in previous commit for bringing in new link option. It should be reverted back to one line format.